### PR TITLE
Add content to v-progress-linear

### LINKED ISSUE #3768: [Feature Request] add ability to show content inside v-progress-linear
Hello!

### Problem
**v-progress-linear** has no ability to show text on the bar, So right now its not possible to have progress bars like these
progress-with-text

### Proposed solution
Please add ability (slot) similar to **v-progress-circular** so it will be possible to add content and align it (left, right, center).

### PR DESCRIPTION
**Pull request by spezzino**
## Description

Adds default slot rendering to v-progress-linear, so content can be added inside the progress bar.

## Motivation and Context

Resolves #3768

## How Has This Been Tested?

visually tested

## Markup:

<details>

```vue
<v-progress-linear value="75" height="20" color="error">
    children
</v-progress-linear>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have created a PR in the [documentation](https://github.com/vuetifyjs/vuetifyjs.com) with the necessary changes.

### CODE REVIEW

File: src/components/VProgressLinear/VProgressLinear.js [hunk 1]
```diff
@@ -109,6 +109,15 @@ export default {
         this.genBar(h, 'long'),
         this.genBar(h, 'short')
       ])
+    },
+    genContent (h) {
+      const children = [this.$slots.default]
```
Comment (jacekkarczmarczyk) [inline src/components/VProgressLinear/VProgressLinear.js]:
```js
      return this.$slots.default && h('div', {
        staticClass: 'v-progress-linear__content'
      }, this.$slots.default)
```

File: src/stylus/components/_progress-linear.styl [hunk 2]
```diff
@@ -46,6 +46,15 @@
     bottom: 0
     transition: .3s ease-in
 
+  &__content
+    width: 100%
+    position: absolute
+    top: 0
+    left: 0
+    bottom: 0
```
Comment (jacekkarczmarczyk) [inline src/stylus/components/_progress-linear.styl]: `left: 0` is not necessary

**Comment by spezzino** [comment]
Changes applied

**Comment by johnleider** [comment]
Please resolve conflicts.

**Comment by nekosaur** [comment]
@spezzino please resolve conflicts and remove irrelevant commits

**Comment by johnleider** [comment]
Closing due to inactivity.

**Pull request closed without merging**